Readability.js

```diff
@@ -2136,6 +2137,29 @@ Readability.prototype = {
           (!isList && weight < 25 && linkDensity > 0.2) ||
           (weight >= 25 && linkDensity > 0.5) ||
           ((embedCount === 1 && contentLength < 75) || embedCount > 1);
+
+        if (haveToRemove) {
+          // Check for lists of images
```

Comment:
Instead of having comments on each line, could we have one comment at the beginning explaining why we are doing this check?